configure-c8y.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+
+if [ `id -u` != 0 ] ; 
+  then printf "Please run as sudo or root!\n"
+  exit
+fi
+printf "\n*********************************************"
+printf "\nThis script lets you setup a thinEdge connection to cumulocity IoT.\nWe assume that the installation was successful and all the necessary services are running.
+Just follow the instructions....."
+printf "\n*********************************************"
+
+printf "\n\n"
+read  -p 'Enter the Cumulocity tenant url e.g. [your-tenant.cumulocity.com]: ' tenant_url
```

Comment:
It would be better to make the script idempotent, so that if the script has stops due to an encountered error, then you can re-run the script and it does not prompt you for the information that you already entered in, or at most the default value is set to the current value if it exists.

Reply:
Will try that in the next version.